pkg/cacheutil/memcached_client.go

```diff
@@ -373,6 +384,10 @@ func (c *memcachedClient) SetAsync(_ context.Context, key string, value []byte,
 }
 
 func (c *memcachedClient) GetMulti(ctx context.Context, keys []string) map[string][]byte {
+	span, _ := tracing.StartSpan(ctx, "memcached_get_multi")
```

Comment:
If I'm not mistaken we used to have this, but this created so many spans that it was impossible to make sense of the resulting trace. I believe @kakkoyun was working on this previously.

Reply:
I can't recall this. I guess you recall the spans related to mutli-tsdb actions. However you have a point we should make sure if this created too many spans.